gr-blocks/lib/add_const_ff_impl.cc

```diff
@@ -43,30 +43,26 @@ add_const_ff_impl::add_const_ff_impl(float k)
 {
 }
 
+void add_const_ff_impl::set_k(float k)
+{
```

Comment:
Put the following here: `gr::thread::scoped_lock guard(d_setlock);`

Reply:
Reason: We don't want to be setting and using either class variable simultaneously (no "both/and" here), so we use a scoped lock to make sure it's "either/or" ...